packages/tcc/build.sh

```diff
@@ -1,20 +1,23 @@
+_COMMIT=4e91d38ddcdb659a4b2c298850c6ccf115073748
+_COMMIT_DATE=20240225
+_COMMIT_SHA256=26974776737d9da876395cc72a005a63526e305aeaa28ebf84c56778a8d9fdac
+
 TERMUX_PKG_HOMEPAGE=https://bellard.org/tcc/
 TERMUX_PKG_DESCRIPTION="Tiny C Compiler"
 TERMUX_PKG_LICENSE="LGPL-2.1"
 TERMUX_PKG_MAINTAINER="@termux"
-_COMMIT=6a24b762d3e1086dcffd002c68cb5ca3a33a5c6d
-_COMMIT_DATE=20230415
-TERMUX_PKG_VERSION=1:0.9.27-p${_COMMIT_DATE}
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION=1:0.9.28-p${_COMMIT_DATE}
```

Comment:
```suggestion
_COMMIT=4e91d38ddcdb659a4b2c298850c6ccf115073748
_COMMIT_DATE=20240225
TERMUX_PKG_VERSION=1:0.9.28-p${_COMMIT_DATE}
```
`_COMMIT` and `_COMMIT_DATE` are usually inlined with the `TERMUX_PKG_*` variables.
For an example see:
https://github.com/termux/termux-packages/blob/b6e127fc33bd9713304d01f3de18f7ddcd6d9a71/packages/tmux/build.sh#L5-L9
Although I don't believe this is a "rule" in the [guidelines](https://github.com/termux/termux-packages/wiki/Coding-guideline).
It's just a convention most other occurences of these variables in build scripts seem to follow.

Reply:
0.9.28 is also a release candidate version not stable version for now. So I suggest you use `TERMUX_PKG_VERSION=1:0.9.28~p${_COMMIT_DATE}` instead.

Reply:
I really think we should move back to archive tarball once 0.9.28 is declared stable. https://download.savannah.nongnu.org/releases/tinycc/

There's no real advantage of using git clone unless need to pull some submodules dependencies.